LambdaEngine/Include/Input/API/InputActionSystem.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include "Input/API/InputState.h"
+#include "Containers/THashTable.h"
+
+namespace LambdaEngine
+{
+	enum EAction {
+		PLAYER_FORWARD,
+		PLAYER_BACKWARD,
+		PLAYER_LEFT,
+		PLAYER_RIGHT,
+		// more actions to come...
+	};
```

Comment:
* This should probably not be a enum since it means that we have to add new entries for every action in the game. A better solution would be a hashmap (std::unordered_map) with a string as key maybe?

* We use the following style with for enums:
`enum EName`
`{ `
    `NAME_ENTRY_ONE = 0,`
    `NAME_ENTRY_TWO = 1`
    `// etc.`
`};`